### ISSUE TITLE
Auto Shots generator. 

### ISSUE THREAD

**Issue by ahmadaIanazi** [shinework/photoshot#21]
I honestly think, most end-users would not be interested in typing Prompts, which they have no idea how it works and what it does. 

The best way to guide the END-USER to have know how prompting works, and what it does is by giving an example and kind of a (gift) by **AUTO GENERATING 10 - 20 well written prompts using their STUDIO NAME.** 

Then, the end-user can use the purchased shots to its best use. 

It would be amazing if we can do the following: 
- Control the number of AUTO prompts.  ( from .env.local )
- Control the auto prompts ARRAY with ${fill_studio_name} here.

Thanks alot.

**Comment by shinework** [comment]
Hello @ahmadaIanazi! That's a great feature, I had also thought of it. Definitely consider implementing it soon. If you have any examples of prompts that have performed well, please share them. Thanks!

**Comment by ahmadaIanazi** [comment]
Very happy that it was on your mind, and I am still testing my version. But I will be documenting the prompts. then, I will be more than happy to share examples, results along with prompts.txt  if I come up with something share worthy.

**Comment by JustinMKY** [comment]
> Very happy that it was on your mind, and I am still testing my version. But I will be documenting the prompts. then, I will be more than happy to share examples, results along with prompts.txt if I come up with something share worthy.

@ahmadaIanazi -- Hello, do you have any update on this feature?

**Comment by ahmadaIanazi** [comment]
@JustinMKY No, I honestly stopped working on it, I got busy with two large projects, but this one always has a place in my heart, that I want to go back to.

**Comment by ahmadaIanazi** [comment]
> > @JustinMKY No, I honestly stopped working on it, I got busy with two large projects, but this one always has a place in my heart, that I want to go back to.
> 
> Alright, could you share code you already have completed?

I am assuming you are looking for the Auto-generating code, which I did not make or got it working at all. As I said I tried but couldn't figure it out. Trust me my code will not even help a little. Sorry dear.
